test/format/verify.c

```diff
@@ -234,6 +235,20 @@ table_verify_mirror(
       table->id, checkpoint == NULL ? "" : checkpoint, checkpoint == NULL ? "" : " checkpoint ");
     trace_msg(session, "%s: start", buf);
 
+    /*
+     * If we are not reading from a checkpoint, start a cursor to pin a page to ensure we never
+     * refresh our snapshot in verification.
+     */
+    if (checkpoint == NULL) {
+        wt_wrap_open_cursor(session, base->uri, NULL, &pinned_cursor);
+        pinned_ret = pinned_cursor->next(pinned_cursor);
+        /* Nothing to verify. */
+        if (pinned_ret == WT_NOTFOUND) {
```

Comment:
You should probably assert pinned_ret is 0 after this so that if there is another error value, we don't try to use it.

Reply:
Never mind. You do have this!